### PR TITLE
made changes that makes the page fit desktop screens using RWD

### PR DESCRIPTION
**Pull request by jossi30**
added some feature to make the mobile version fit to larger screens using responsive web design

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -385,3 +416,206 @@ button:hover {
...
+  }
+
+  .project-detail {
+    width: 480px;
+    height: 400px;
+    padding-top: 25px;
+    display: flex;
+    flex-direction: column;
+    align-items: baseline;
+  }
```
Comment (mckent05) [inline style.css:518]: - There are some little differences between the project card in your desktop design and the one in the Figma template. Please fix this for all your project cards as they may apply. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your design -
- Please pay attention to the following
-- The tech stack design (HTML, CSS, Javascript) is not the same as it is in the design
-- The `See project` button is not the same as it is in the design
-- The project cards have a border around them.
roject card youisef

Figma design -
project card project

File: style.css [hunk 2]
```diff
@@ -385,3 +416,206 @@ button:hover {
...
+
+  .menu-button-burger {
+    display: none;
+  }
+
+  .nav-menu {
+    padding-left: 550px;
+    display: flex;
+    gap: 10px;
+  }
```
Comment (mckent05) [inline style.css:444]: - Good job on the work done but your nav doesn't exactly match the Figma design. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your design (please pay attention to the margins I have alighted. The margins aren't equal. Please try to fix this)-
nav youisef

Figma template -
nav figma1

File: style.css [hunk 3]
```diff
@@ -385,3 +416,206 @@ button:hover {
...
+  .project-info {
+    width: 450px;
+  }
+
+  .about {
+    flex-direction: row;
+    margin: 100px 140px;
+    gap: 60px;
+    width: 90vw;
+  }
```
Comment (mckent05) [inline style.css:548]: - There are slight differences between your design for the `about myself` section and the given Figma template. Please ty to fix this.

Your design(Pay attention to the alignment of the arrows) -
tech stack youisef

Figma template-
tech stack desktop

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @jossi30 , 

# STATUS : CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- Your portfolio is looking really great 👍 👍 
- Descriptive PR title and summary 💯 💯 
- All linter checks are passing ✔️ ✔️ 


## Required Changes ♻️
- Please check the mobile version of your portfolio, you have a bug in the about myself section.  Please try to fix this 🙏 🙏 

Your design:
youisef mobile

_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -34,50 +40,53 @@ <h1 class="hello">I’m Yosief<br>Glad to see<br>you!</h1>
...
-                <p class="project-info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                <ul class="project-lang">
-                    <li class="lang-name">html</li>
-                    <li class="lang-name">css</li>
-                    <li class="lang-name">javascript</li>
+                <p class="project info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="project-langs">
+                    <li class="lang name">html</li>
+                    <li class="lang name">css</li>
+                    <li class="lang name">javascript</li>
```
Comment (roshan-bajgain) [inline index.html:61]: - [ ] I would suggest you please, fix that the page is divided half of it contains the project's picture and the other half contains the project's description. As requested by the previous reviewer please fix the spacing by seeing the design
- [ ] please fix the buttons as well
### Figma design
figma
### Your design
your

File: index.html [hunk 5]
```diff
@@ -17,15 +17,21 @@
             <div class="logo">My Logo</div>
             <div class="menu-button">
                     <div class="menu-button-burger"></div>
+                        <div class="nav-menu">
+                            <a class="menu-item" href="#portfolio">Portfolio</a>
+                            <a class="menu-item" href="#about">About</a>
+                            <a class="menu-item" href="#contact">Contact</a>
+                          </div>
```
Comment (roshan-bajgain) [inline index.html:24]: - [ ] If you see the nav links then It is aligned at right so please fix it. See the design for reference
### Figma design
figma
### Your design
your

File: index.html [hunk 6]
```diff
@@ -158,17 +135,20 @@ <h3 class="project-name">Multi-Post<br>Stories</h3>
...
-        <li class="frame"><a href="#"></a> <img src="img/Frame 10.svg" alt="html"></a></li>
-        <li class="frame"><a href="#"></a> <img src="img/Frame 11.svg" alt="css"></a></i></li>
+    </div>    
+    <section>   
+        <div><li class="list-items">Languages<i class="arrow down"></i></li></div>
+        <div class="lang-details">
+        <div class="lang-container"><li class="lang-name-tech"><a href="#"></a> <img src="img/js.svg" alt="js"></a>Javascript</li></div>
+        <div class="lang-container"><li class="lang-name-tech"><a href="#"></a> <img src="img/html.svg" alt="html">Html</a></li></div>
+        <div class="lang-container"><li class="lang-name-tech"><a href="#"></a> <img src="img/css.svg" alt="css">Css</a></i></li></div>
+        </div>
```
Comment (roshan-bajgain) [inline index.html:146]: - [ ] There are 2 blocks The left one takes 40% and the right one the remaining 60% of the page's width, and the contents of each of them are aligned to the right. please see the design below and make changes
### Figma design
figma
### Your design
your

Comment (roshan-bajgain) [inline index.html:146]: - [ ] As requested by the previous reviewer, please make your about section match the design. As both, the content is aligned at the same line and there is more spacing on the right side
### your mobile design
yourmbl
### Fimga of the mobile version
figma

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @jossi30, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @jossi30, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

-   No linter errors. :heavy_check_mark: 
-   Good README. :heavy_check_mark: 
-   Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**